libvmi/os/linux/core.c

```diff
@@ -433,6 +434,16 @@ void linux_read_config_ghashtable_entries(char* key, gpointer value,
     _done: return;
 }
 
+status_t linux_get_kernel_struct_offset(vmi_instance_t vmi, const char* symbol, const char* subsymbol, addr_t *addr){
+    status_t ret = VMI_SUCCESS;
+    linux_instance_t linux_instance = vmi->os_data;
+    if (VMI_FAILURE == rekall_profile_symbol_to_rva(linux_instance->rekall_profile,symbol,subsymbol,addr)){
```

Comment:
You can just return the call to rekall_profile_symbol_to_rva directly, no need, for the local variable ret.